RELEASE.md

```diff
@@ -1,3 +1,6 @@
+# Release 1.1
+ - Added inter-domain inducing features. Inducing points are used by default and are now set with `model.feat.Z`.
+
 # Release 1.0
```

Comment:
Would it be better to have changes to Travis file in separate PR( if actually needed)?

Reply:
They are needed, as the tests are failing here because of configuration issues, not because of the code (tests run fine locally). This could be done in a separate PR as well.